app/models/data_importer.rb

```diff
@@ -0,0 +1,35 @@
+require 'json'
+require "sinatra/activerecord"
+require_relative './well_inspection'
+
+class DataImporter
+
+  def init
+    save_objects
+  end
+
+  # for now get from data.json
+  # eventually hit the API endpoint and
+  # 1. parse headers to create table
+  # 2. build migration
+  # 3. run migration
+  # 4. parse json for data array
+  # 5. store json objs as ar instances
```

Comment:
This is the general flow I envisioned for getting data into pg. This would be a general "given a resource endpoint, get all data and store it as is".

 I think as a second iteration, the process should include a custom filtering method / module (set uniqueness constraints, remove an unused column, etc). So, using the numbers above, the process would go `1, 4, X, 2, 3, 5`, where X is the filtering.

Is that feasible or off track from the end goal?